docs/source/column.rst

```diff
@@ -63,7 +64,7 @@ Column class reference
     Column.to_arrow
     Column.to_tensor
     Column.to_pylist
-    Column.to_pandas    
+    Column.to_pandas
```

Comment:
separate to this PR -- looks like the current Pandas conversion is not efficient : 

https://github.com/pytorch/torcharrow/blob/main/torcharrow/icolumn.py#L1104-L1106

Maybe we should use something like `self.to_arrow().to_pandas()` to fix it? :)